pages/api/email/sign-up.ts

```diff
@@ -0,0 +1,35 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import { signUpForMailingList } from '../../../lib/api/emailLogic';
+import makeRoute from '../../../lib/utils/utils';
+
+/**
+ * @swagger
+ * /api/email/sign-up:
+ *   post:
+ *     description: Send email to partner to confirm sign-up
+ *     parameters:
+ *       - name: firstName
+ *         required: true
+ *         description: First name of individual.
```

Comment:
❓ Do these really need descriptions if each on is just `<the thing> of individual`?

Reply:
Prob not but that's more so just placeholder text until the actual route logic is built